extensions/query-history/package.json

```diff
@@ -46,9 +45,13 @@
 			]
 		}
 	},
-	"dependencies": {
-	},
+	"dependencies": {},
 	"devDependencies": {
 		"vscode": "1.0.1"
+	},
+	"__metadata": {
+		"id": "55",
+		"publisherDisplayName": "Microsoft",
+		"publisherId": "Microsoft"
 	}
-}
+}
```

Comment:
nit: I'm not sure what our standard for jsons are but we have a new line in all of our ts files at the end of the document

Reply:
Yeah I don't think the formatter picks that up but I'll add one anyways to be consistent. Thanks!